addons/pinniped/config-controller/controllers/controllers.go

```diff
@@ -59,6 +59,10 @@ func (c *PinnipedController) SetupWithManager(manager ctrl.Manager) error {
 	return nil
 }
 
+// +kubebuilder:rbac:groups="",resources=secrets,verbs=list;watch;get;patch;update;delete
```

Comment:
q: do we not need `delete` anymore because we are letting the `Secret` get deleted via its `OwnerRef`? cc @sabbey37

Reply:
yeah, we shouldn't need delete any longer either!

Reply:
removing `delete` 👍